content/docs/contributing/to-docs/o3de-docs-repo-setup.md

```diff
@@ -146,14 +146,13 @@ Hugo is the static site builder that O3DE documentation uses. When you have Hugo
 
 1. Get the **extended** Hugo binary. For Hugo installation, refer to [Install Hugo](https://gohugo.io/getting-started/installing/). The extended binary is required for some of the features that the O3DE documentation site uses.
 
-2. You need to add the Node.js bootstrap package. Bootstrap contains some modules that are used to style the O3DE documentation site. Get Node.js here [Download Node.js](https://nodejs.org/en/download/) and run the installer.
+1. To install **npm**, follow the instructions in the [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) documentation. Installing npm also installs **Node.js**.
 
-3. In the terminal `cd` to the root of your o3de.org clone.
-
-4. Use `npm` to install bootstrap with the command below:
+1. To install dependencies, run the following command from the `o3de.org` repository:
 
     ```shell
-    npm install bootstrap
+    cd <path-to-repo>/o3de.org
+    npm install
     ```
 
 ### Run a Hugo server
```

Comment:
I just noticed that step 3 in "Run a Hugo server" (which I can't highlight in this PR) says that a server usually starts on port `1333`. This should be `1313`.

Also, consider adding a step 4 that shows an example of using this port to view the local build in a web browser, as shown in the readme.